### ISSUE TITLE
request for comment: DiSSCo specimen & collection classification

### ISSUE THREAD

**Issue by wouteraddink** [tdwg/cd#470]
For your information, an RFC for a high level specimen and collection classification, to implement in DiSSCo and to use in the FDO record for a digital specimen. Since this is related to how we plan to implement LtC, I also post it here.
[RFC Document](https://docs.google.com/document/d/19OPyOm9VF2qfI3M6RmJPvRfo8JlZ3tt0II05aGCyBHQ)
please review this RFC before 15 April 2023. Note that controlled vocabularies are not part of the LtC standard in development.

**Comment by Jegelewicz** [comment]
It is unfortunate that @tdwg/material-sample work is not considered in this at all. It might benefit those working on the document to participate in the MaterialSample Task Group.

**Comment by Jegelewicz** [comment]
dwc: materialSampleType - as far as I know, there is no such term in Darwin Core?

**Comment by tucotuco** [comment]
That is correct, the term dwc:materialSampleType is a fabrication.

**Comment by Jegelewicz** [comment]
In addition, the MaterialSample Task Group has determined that a term such as materialSampleType would be overburdened and we have been considering the iSamples terms (yes all three). But we definitely see issues with the controlled vocabularies in iSamples as they relate to biological material and we have had to move away from this in order to work on a larger class of material - MaterialEntity as we attempt to create appropriate material terms for the new GBIF model that is in testing.

**Comment by Jegelewicz** [comment]
> MIDS and LtC are ready enough to build on

While this may be true, as far as I know these have not been formally adopted and could end up being different than whatever is there now.

**Comment by wouteraddink** [comment]
That is correct and we are aware of it. In that case we may need to adjust
implementation.





Op do 23 mrt. 2023 19:54 schreef Teresa Mayfield-Meyer <
***@***.***>:

**Comment by wouteraddink** [comment]
OK, so it looks we are in agreement?

Op do 23 mrt. 2023 19:52 schreef Teresa Mayfield-Meyer <
***@***.***>:

**Comment by wouteraddink** [comment]
Correct, but was it not discussed as a possible new DwC term? That is what
I understood from the GitHub issues.  If not then there is no problem.

Op do 23 mrt. 2023 19:42 schreef Teresa Mayfield-Meyer <
***@***.***>:

**Comment by tucotuco** [comment]
Yes, there is an open issue for it, which was relegated to the Material
Sample Task Group.
